core.js

```diff
@@ -706,11 +709,11 @@ export class FileTypeParser {
 			try {
 				await tokenizer.ignore(1350);
 				const maxBufferSize = 10 * 1024 * 1024;
-				const buffer = Buffer.alloc(Math.min(maxBufferSize, tokenizer.fileInfo.size));
+				const buffer = new Uint8Array(Math.min(maxBufferSize, tokenizer.fileInfo.size));
 				await tokenizer.readBuffer(buffer, {mayBeLess: true});
 
 				// Check if this is an Adobe Illustrator file
-				if (buffer.includes(Buffer.from('AIPrivateData'))) {
+				if (indexOf(buffer, new TextEncoder().encode('AIPrivateData')) >= 0) {
```

Comment:
```suggestion
				if (indexOf(buffer, new TextEncoder().encode('AIPrivateData')) !== -1) {
```

Reply:
From this usage, it looks like it would be useful to have a `includes` method in `uint8array-extras` too.